index.js

```diff
@@ -103,7 +110,7 @@ function initApp(callback) {
     /*
      * configure routes
      */
-    app.get('/api/v1/...', controllers...);
+    app.post('/api/v1/public-share', controllers.download.create);
```

Comment:
The endpoint should actually be `/api/v1/compendium`, since that is what happens here, a new compendium is created. I've missed that in the docs before, please change.